projects/batavierenrace/src/main.ts

```diff
@@ -13,14 +13,11 @@ bootstrapMediaApp({
 
   apiHosts: {
     // Development (local)
-    "localhost": "api.dev.kick-in.media",
-
-    // Testing
-    "dev.kick-in.media": "api.dev.kick-in.media",
+    "localhost": "api.foto.batavierenrace.nl",
```

Comment:
Localhost (de ontwikkelomgeving) laat ik wel graag op de kick-in dev api staan. 
```suggestion
    "localhost": "api.dev.kick-in.media",
```